book/src/access_control/intro.md

```diff
@@ -69,6 +70,7 @@ tasks within Kanidm as required.
 | `idm_people_on_boarding`     | create (but not modify) persons. Intended for use with service accounts |
 | `idm_people_pii_read`        | allow read to personally identifying information                        |
 | `idm_people_self_write_mail` | allow self-modification of the mail attribute                           |
+| `idm_people_self_name_write` | allow self-modification of the name related attributes                  |
```

Comment:
We've got different naming conventions here... I can't live with that inconsistency 😄 

Should be `idm_people_self_write_name` given the current naming ... then again we've got `pii_read` ... so I kinda think renaming `write_mail` is better than renaming `name_write` 🤣

Reply:
We can rename them for free basicly, it doesn't break anything. I'll change them to be consistent. I'd say we should make the pattern "idm_<class>_<target>_<action>". So that would make it "idm_people_read_pii", "idm_people_self_name_write", "idm_people_self_mail_write". What do you think? Alternately, swap target and action and we just update "idm_people_self_mail_write"

Reply:
Yep, `idm_<target>_<verb>` where `<target>` narrows from largest scope to smallest, so `self_mail` or `all_persons_name` etc. Probably should document that, too.

Reply:
`idm_people_pii_read` though 😄